### ISSUE TITLE
No assets when deploying to Heroku 

### ISSUE THREAD

**Issue by kevinchau321** [kriasoft/react-starter-kit#1138]
I'm trying to deploy to heroku using `tools/deploy.js'.

I know I set up the remote correctly because I can use it to push my app to heroku but none of the files in `build/public` or `build/public/assets` are making it into my heroku git repo. This results in failing to load images or vendor scripts.

I can confirm this by pulling from heroku's git and I see that the `public` folder is empty. Is there something I might be missing when using the deploy script?

Heroku App: https://treactr.herokuapp.com/

**Comment by frenzzy** [comment]
Remove git folders inside build folder and try again:
`rm -rf build && yarn run deploy`

**Comment by kevinchau321** [comment]
Hmm tried that, but no luck, using the deploy script is still not pushing the public folder to heroku app.

**Comment by kevinchau321** [comment]
When I see what git is tracking in the build directory it looks like it is tracking the public folder but none of its contents:

```
➜  build git:(master) git ls-tree -r master --name-only
LICENSE.txt
assets.json
package.json
public
server.js
server.js.map
```

**Comment by kevinchau321** [comment]
Fixed the problem by manually tracking/adding all the contents of the public folder. My command line looked something like this:

```
$ mv public public2
$ git status
$ git add .
        deleted:    public
        new file:   public2/tile.png
        ...
$ mv public2 public
$ git add .
$ git status
        deleted:    public2
        new file:   public/apple-touch-icon.png
       ...
```

Then I committed and manually pushed to heroku. Finally my assets are in the heroku app. From then on I was able to deploy using the deploy.js script. Seems like there should be a git command of some kind that I am not thinking of to manually add all those files, but either way I am not at all sure how this bug came up in the first place.

It might have something to do with the fact that I originally tried to deploy a static site to gh-pages with the script before switching to deploying to Heroku, but I'll close the issue for now. Thanks.
